LocalSettings.php

```diff
@@ -1772,6 +1767,16 @@
 			'Varnish' => 'https://meta.miraheze.org/wiki/Tech:Varnish',
 		],
 	],
+	'wgIncidentReportingInactiveServices' => [
```

Comment:
Parsoid and Bacula should be moved to inactive, and I do not think ElasticSearch should be.